web-ui/src/pages/AnniversaryReportPage.test.jsx

```diff
@@ -1,16 +1,16 @@
 import React from "react";
```

Comment:
This test file was renamed to indicate that it tests the new `AnniversaryReportPage` component.